app/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
 dependencies {
 
     // Support libraries.
-    compile 'com.android.support:appcompat-v7:24.0.0'
+    compile 'com.android.support:appcompat-v7:24.2.1'
```

Comment:
This is required to fix a bug with FileProvider pointing incorrectly to the the external-files directory